src/components/Contact/ContactForm.module.css

```diff
@@ -0,0 +1,17 @@
+body{
+    background-color: var(--midnight-green);
+}
+
+.ContactForm{
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    gap: 24px;
+}
+
+.ContactFormInputs{
```

Comment:
Since bothe tye styles are same better to use a reusable name and not repeat the css

Reply:
renamed and removed repeation.